mobile/Cargo.toml

```diff
@@ -12,6 +12,8 @@ tokio = { version = "1.37.0", features = [
     "rt-multi-thread",
 ] }
 log = "0.4.21"
+openssl = { version = "0.10.64", features = ["vendored"] }
```

Comment:
Why is openssl added as dependency here? If it is only to have it vendored, the `vendored-openssl` feature flag should be enabled on the mycelium dependency

Reply:
>  the vendored-openssl feature flag should be enabled on the mycelium dependency

oh, didn't know this feature.

Something like this?
```toml
mycelium = { path = "../mycelium", features = ["vendored-openssl"]}
```

Reply:
fixed